helm/examples/efs/associate-oidc.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+WORK_DIR=$(dirname $0)
+
+source ${WORK_DIR}/efs-env
+
+oidc_id=$(aws eks describe-cluster --name ${EKS_NAME} --query "cluster.identity.oidc.issuer" --output text | cut -d '/' -f 5)
+echo "oidc_id: $oidc_id"
+oidc_providers=$(aws iam list-open-id-connect-providers | grep $oidc_id)
+if [ "${oidc_providers}" = "" ]
```

Comment:
trivial: more common to test empty with -z :
```
if [ -z "${oidc_providers}" ]
```

Reply:
fixed